waiter/test/waiter/mesos/marathon_test.clj

```diff
@@ -14,25 +14,30 @@
 ;; limitations under the License.
 ;;
 (ns waiter.mesos.marathon-test
-  (:require [clojure.string :as str]
-            [clojure.test :refer :all]
+  (:require [clojure.test :refer :all]
             [waiter.mesos.marathon :refer :all]
             [waiter.util.http-utils :as http-utils]))
 
+(def ^:private http-client (Object.))
+
+(def ^:private marathon-url "http://www.marathon.com:1234")
```

Comment:
This is a valid URL.  It would be best to avoid real URLs to prevent accidental traffic.